src/Core/Asset/GeometryData.cpp

```diff
@@ -50,17 +41,33 @@ void GeometryData::displayInfo() const {
     case HEX_MESH:
         type = "HEX MESH";
         break;
+    case UNKNOWN:
+    default:
+        type = "UNKNOWN";
+        break;
     }
+
+    auto attribSize = [this]( Geometry::MeshAttrib a ) -> size_t {
+        const auto& name = getAttribName( a );
+        return getGeometry().hasAttrib( name ) ? getGeometry().getAttribBase( name )->getSize() : 0;
```

Comment:
```suggestion
        return getGeometry().hasAttrib( getAttribName( a ) ) ? getGeometry().getAttribBase( name )->getSize() : 0;
```

Reply:
I'm not sure to see the difference. Either  `getAttribName( a )` is called twice (your proposal) or once (the actual implementation) and cached in a temporary. I prefer the latter. What is the advantage of the former ?

Reply:
right, I do not see the second use of name, but my proposal is to be consistent between the two lambdas.